workspace_tools/toolchains/gcc.py

```diff
@@ -69,11 +69,8 @@ def __init__(self, target, options=None, notify=None, macros=None, silent=False,
         if "save-asm" in self.options:
             common_flags.append("-save-temps")
 
-        if "debug-info" in self.options:
-            common_flags.append("-g")
-            common_flags.append("-O0")
-        else:
-            common_flags.append("-O2")
+        common_flags.append("-ggdb")
```

Comment:
Yes, please. If you consider ggdb as an useful addition , please send additional PR. Thanks